content/centralized.md

```diff
@@ -49,47 +49,63 @@ Real life examples:
-- For online teaching, use breakout rooms.
-- **One person per group (administrator)** generates a new repository
-  from the template <https://github.com/coderefinery/template-centralized-workflow-exercise>
-  called `centralized-workflow-exercise` (There is no need to tick *"Include all branches"* for this exercise):
-  ```{figure} img/centralized/generate_repo.png
-  :alt: Screenshot of generating the exercise repository
-  :width: 100%
-  ```
-- Then **everyone in your group** needs their GitHub account to be added as collaborator to the exercise repository:
-  - Participants give their GitHub usernames to their chosen administrator (in their respective group, in online workshops you can use the Zoom chat for private communication within the breakout room).
-  - Administrator gives the other group members the newly created GitHub repository URL.
-  - Administrator adds participants as collaborators to their project (Settings → Manage Access → Invite a collaborator).
-  - Group members need to accept the invitation. GitHub emails you an invitation link, but if you don't receive it
-    you can go to your GitHub notifications in the top right corner. The administrator can also "copy invite link"
-    and share it within the group.
-
-- Watching and unwatching repositories:
+``````{prereq} Exercise preparation
+`````{tabs}
+  ````{tab} Part of team/ exercise room
+  - In-person: We form small groups (4-5 persons). Video: use breakout rooms.
+  - Each group needs to appoint someone who will host the shared GitHub repository: *an administrator*.
+    This is typically the exercise lead (if available).
+  - **One person per group (administrator)** generates a new repository
+    from the template <https://github.com/coderefinery/template-centralized-workflow-exercise>
+    called `centralized-workflow-exercise` (There is no need to tick *"Include all branches"* for this exercise):
+    ```{figure} img/centralized/generate_repo.png
+    :alt: Screenshot of generating the exercise repository
+    :width: 100%
+    ```
+  - Then **everyone in your group** needs their GitHub account to be added as collaborator to the exercise repository:
+    - Participants give their GitHub usernames to their chosen administrator (in their respective group, in online workshops you can use the Zoom chat for private communication within the breakout room).
+    - Administrator gives the other group members the newly created GitHub repository URL.
+    - Administrator adds participants as collaborators to their project (Settings → Manage Access → Invite a collaborator).
+  ````
+
+  ````{tab} Participating via stream
+  We create(d) these the day before hopefully:
+  - <https://github.com/cr-workshop-exercises/centralized-workflow-exercise> (this will not be shown on stream)
+  - <https://github.com/cr-workshop-exercises/centralized-workflow-exercise-recorded> (this will be shown on stream and recorded)
```

Comment:
```suggestion
  - <https://github.com/cr-workshop-exercises/centralized-workflow-exercise-recorded> (this will be shown on stream and recorded, **your username and comments may appear in the recorded video on YouTube**)
```